main.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from PIL import Image
+from pathlib import Path
+from random import randint
+
+
+def getimages(location):
+    image = []
```

Comment:
name should be sonething like images since it's a list. something that tells you it's multiple of those

Reply:
done